README_ru-RU.md

```diff
@@ -0,0 +1,447 @@
+ababab Adiciona o método `use` ao model Credit
+efefef Usa o InventoryBackendPool para recuperar o backend de estoque
+bebebe Corrige nome de método na classe InventoryBackend
+```
+
+```
+# Плохо (смесь Английского и Португальского)
+ababab Usa o InventoryBackendPool para recuperar o backend de estoque
+efefef Add `use` method to Credit model
+cdcdcd Agora vai
+```
+
+### Шаблон
+
+Этот шаблон, [который изначально описан Tim Pope](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html), также появляется в [_Pro Git Book_](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project).
+
+```
+Summarize changes in around 50 characters or less
+
+More detailed explanatory text, if necessary. Wrap it to about 72
+characters or so. In some contexts, the first line is treated as the
+subject of the commit and the rest of the text as the body. The
+blank line separating the summary from the body is critical (unless
+you omit the body entirely); various tools like `log`, `shortlog`
+and `rebase` can get confused if you run the two together.
+
+Explain the problem that this commit is solving. Focus on why you
+are making this change as opposed to how (the code explains that).
+Are there side effects or other unintuitive consequences of this
+change? Here's the place to explain them.
+
+Further paragraphs come after blank lines.
+
+ - Bullet points are okay, too
+
+ - Typically a hyphen or asterisk is used for the bullet, preceded
+   by a single space, with blank lines in between, but conventions
+   vary here
+
+If you use an issue tracker, put references to them at the bottom,
+like this:
+
+Resolves: #123
+See also: #456, #789
+```
+
+## Перебазировка vs. Слияние
+
+Эта секция является кратким пересказом превосходного руководства Atlassian, ["Merging vs. Rebasing"](https://www.atlassian.com/git/tutorials/merging-vs-rebasing).
+
+![](https://wac-cdn.atlassian.com/dam/jcr:01b0b04e-64f3-4659-af21-c4d86bc7cb0b/01.svg?cdnVersion=hq)
+
+### Перебазировка
+
+**Кратко:** Применяет ваше ответвление коммитов, один за другим, к базовой ветви, генерируя новое дерево. 
+
+![](https://wac-cdn.atlassian.com/dam/jcr:5b153a22-38be-40d0-aec8-5f2fffc771e5/03.svg?cdnVersion=hq)
+
+### Слияние
+
+**Кратко:** Создаёт новый коммит, называемый (соответственно) _слияющим коммитом_ (_merge commit_), с различиями между двумя ветвями.
+
+![](https://wac-cdn.atlassian.com/dam/jcr:e229fef6-2c2f-4a4f-b270-e1e1baa94055/02.svg?cdnVersion=hq)
+
+### Почему некоторые люди предпочитают перебазирование вместо слияния?
+
+Я, в частности, предпочитаю перебазирование вместо слияния. Причины включают в себя: 
+
+* Это генерирует "чистую" историю, без ненужных слияющих коммитов
+* _Что видишь - то и получаешь_, т.е. при обзоре кода все изменения идут от определенного коммита, избегая скрытые изменения в слияющих коммитах 
+* Больше слияний решаются автором коммита, и каждое слитое изменение находится в коммите с соответствующим сообщением
```

Comment:
Need help with translation this line:
* More merges are resolved by the committer, and every merge change is in a commit with a proper message